README.md

```diff
@@ -12,7 +12,7 @@ The matrix below assumes the user always install the latest patch release of the
 
 | Plugin | AiiDA | Python | Quantum ESPRESSO |
 |-|-|-|-|
-| `v4.8 < v5.0` | ![Compatibility for v4.0][AiiDA v4.0-pydantic2] |  [![PyPI pyversions][Python v3.9-v3.12]](https://pypi.org/project/aiida-quantumespresso/4.8.0/) | ![Quantum ESPRESSO compatibility][QE v6.6-7.4] |
```

Comment:
Unfortunately the badges aren't that simple, see

https://github.com/Minotakm/aiida-quantumespresso/tree/Add_support_QE_v7.4.1

You also have to update them at the end of the Markdown file. ^^

Also, this change would only be released in `aiida-quantumespresso` v4.11, so I think we need to add another line to the compatibility matrix, no?

Reply:
It will get bigger and bigger though... Maybe we should move a part to the documentation somewhere, and only leave the 4+ versions. I'll have a think.

Reply:
Yup, that's also what I was thinking. ^^

Reply:
Opened https://github.com/aiidateam/aiida-quantumespresso/issues/1092 to track this, don't want to slow down this PR any further.